tests/foreman/ui/test_settings.py

```diff
@@ -210,30 +209,45 @@ def test_positive_update_login_page_footer_text(session, setting_update):
         2. Click on general tab
         3. Input long length string into login page footer field
         4. Assert value from login page
-        5. Input empty string into the login page footer field
-        6. Assert empty value from login page
+        5. Input default value into the login page footer field
+        6. Assert default value from login page
+        Z. Input empty string into the login page footer field
```

Comment:
Is that Z instead of 7 :-O

Reply:
That's to test if reviewers are attentive enough :D

Reply:
to my defense 
![image](https://github.com/SatelliteQE/robottelo/assets/8287578/ecdf38e5-cfc6-47e2-84b2-65acd981095b)